### ISSUE TITLE
`D` in visual block mode cancels multi-cursor

### ISSUE THREAD

**Issue by J-Fields** [VSCodeVim/Vim#4683]

**Comment by sql-koala** [comment]
Hi @J-Fields I did take a look at this. For me, vscodevim Visual Block  (C-v) behaves like vim, regarding D (and C, #4684 ).
Can you check again?

**Comment by J-Fields** [comment]
Yeah it's still an issue. If you have multiple cursors in visual block mode (meaning multiple blocks), `D` will collapse down to a single cursor, rather than a single cursor per block. This is due to how we represent visual block mode, and I've got an idea of how to address it, but it'll require some substantial re-architecture.

**Comment by sql-koala** [comment]
Ah, yes, you are right. I did not understand the scenario of multiple visual blocks started at multiple cursors.
